addons/account/views/account_move_views.xml

```diff
@@ -645,6 +645,16 @@
                                 attrs="{'invisible' : [('to_check', '=', False)]}" data-hotkey="k" />
                         <field name="state" widget="statusbar" statusbar_visible="draft,posted"/>
                     </header>
+                    <div class="alert alert-warning mb-0" role="alert"
+                         attrs="{'invisible': ['|', ('state', '!=', 'draft'), ('duplicated_ref', '=', [])]}">
+                        Warning: Duplicated bill:
```

Comment:
Should I do something to ensure it gets translated?

Reply:
No, but the opposite can be done if you don't want it to be translated

Reply:
> No, but the opposite can be done if you don't want it to be translated

Just for my information how should I do if I wanted to ?

Reply:
In qweb: `t-translation="off"`
I think there was something else for other view, but I don't remember and can't find it